examples/plugins/test-plugin/main.go

```diff
@@ -32,18 +32,39 @@ type pluginImplementation struct {
 
 var impl = pluginImplementation{}
 
+type FuseConfig struct {
+	DevFuseFd  int
+	MountPoint string
+	Program    []string
+}
+
+type pluginConfig struct {
+	Fuse FuseConfig
+}
+
 func (p pluginImplementation) Initialize(r pluginapi.HookRegistration) {
 	// Adding a custom flag to the action commands
 	flag := pluginapi.StringFlagHook{
 		Flag: pflag.Flag{
 			Name:      "test-flag",
 			Shorthand: "z",
 			Usage:     "test mc testface",
-			DefValue:  "TEST_VALUE_DEFAULT",
 		},
 		Callback: func(f *pflag.Flag, cfg *singularity.EngineConfig) {
 			fmt.Println("Calling back into plugin!")
 			fmt.Printf("Received flag value: %s\n", f.Value.String())
+
+			config := pluginConfig{
+				Fuse: FuseConfig{
+					MountPoint: f.Value.String(),
+					Program:    []string{"/usr/local/bin/fuse-example"},
+				},
+			}
+			if err := cfg.SetPluginConfig("test-plugin", config); err != nil {
+				fmt.Printf("Cannot set plugin configuration: %+v\n", err)
```

Comment:
Should this print to stderr?

```suggestion
				fmt.Fprintf(os.Stderr, "Cannot set plugin configuration: %+v\n", err)
```